charts/lunar-proxy/values.yaml

```
@@ -35,6 +34,11 @@ lunarHubURL: null
 telemetryEnabled: true
 lunarManaged: false
 
+# Lunar Cluster configuration
+lunarCluster:
+  lunarClusterStaleThresholdPeriodInSec: 120
```

```suggestion
  staleThresholdPeriodInSec: 120
```
You already created a parent field `lunarCluster`, don't repeat